types/ip-regex/index.d.ts

```diff
@@ -1,12 +1,19 @@
-// Type definitions for ip-regex 2.0
+// Type definitions for ip-regex 3.0
 // Project: https://github.com/sindresorhus/ip-regex
-// Definitions by: Daniel Perez Alvarez <https://github.com/unindented>
+// Definitions by: Renārs Vilnis <https://github.com/renarsvilnis>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/** Regular expression for matching IP addresses (IPv4 & IPv6). */
-declare function ipRegex(options?: {
-    /** Only match an exact string. Useful with RegExp#test to check if a string is an IP address. */
-    exact?: boolean;
-}): RegExp;
+// tslint:disable-next-line no-single-declare-module no-declare-current-package
+declare module "ip-regex" {
+    function ipRegex(options?: ipRegex.IpRegexOptions): RegExp;
```

Comment:
the source calls this function `ip`. It doesn't matter if the name is different, but it's more confusing to me.

Reply:
Fixed